src/module-2/bootstrap-design/index-bootstrap.html

```diff
@@ -0,0 +1,552 @@
+                                    <label class="form-check-label" for="flexCheckChecked">
+                                        Speck
+                                    </label>
+                                </div>
+
+                                <button type="button" class="btn btn-default w-100 text-light" 
+                                style="background: #6F64F8;">
+                                    Clear all filtres
+                                </button>
+                            </div>
+                        </div> 
+                    </nav>
+                </div>
+
+                <div class="col-lg-9 col-md-12">
+                    <div class="row mb-3">
+                        <div class="col-xl-11 col-md-10">
+                            <span>7,618 results found</span>
+                        </div>
+
+                        <div class="col-xl-1 col-md-2 d-flex justify-content-end">
+                            <button type="button" class="btn btn-primary" style="background: #6F64F8;">
+                                <img src="img/heart.svg"  alt="" width="30" height="24" class="d-inline-block align-text-top">
+                            </button>
+                        </div>
+                    </div>
+
+                    <div class="row mb-3">
+                        <div class="col-xl-11 col-md-10">
+                            <div>
+                                <input type="search" class="form-control" id="exampleFormControlInput1" aria-label="Search" placeholder="Search">
+                              </div>
+                        </div>
+
+                        <div class="col-xl-1 col-md-2 d-flex justify-content-end">
+                            <button type="button" class="btn btn-primary" style="background: #6F64F8;">
+                                <img src="img/4801315_glass_magnifier_magnifying_search_searching_icon.svg"  
+                                    alt="" width="30" height="24" class="d-inline-block align-text-top">
+                            </button>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-xl-4 col-lg-6 col-md-6 mb-4">
+                            <div class="card" style="height: 45vh;">
+                                <img src="img/apple-watch.png" class="img-fluid my-2 mx-auto" 
+                                    alt="..." style="max-height: 50%; max-width: 90%;">
+                                <div class="card-body p-0 position-relative">
+                                    <div class="d-flex justify-content-between mx-3">
+                                        <button type="button" class="btn btn-primary" style="background: #6F64F8;"> 
+                                            3.4 <i class="bi bi-star"></i>
+                                        </button>
```

Comment:
every tag should be from a new line, and I like to have all raw text should be wrapped by some element(`div`, `span`, `p`, `h` ...)
```
<button type="button" class="btn btn-primary" style="background: #6F64F8;"> 
    <div>3.4</div>
    <i class="bi bi-star"></i>
</button>
```